### DISCUSSION TITLE
Mobile UX

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [Schneegans/Fly-Pie#359] [Ideas & Feature Discussions]
I have an idea.

We often use the phone by swiping from the bottom of the screen to open windows or return to the home screen.

BUT
New User Experience may look like:
Swipe from the bottom of the screen up to... 🤔.. Maybe... 250px then open Fly-pie manager!

I think It can be use as the main element of the New Mobile user experience!

**Comment by Schneegans** [comment]
Yeah, this would be cool indeed! Have you checked out my successor project [Kando](https://github.com/kando-menu/kando)? This is cross platform, so you can have something like this on a Linux or Windows tablet, I suppose. However, it does not support mobile for now...

**Reply by Dolbaeb777K** [reply]
Yeah. That's cool

Flypie (kando) should be used only on tablet, because It may uncomfortable experience in mobile phone...
